src/utils/dataPlane-utils.ts

```diff
@@ -240,6 +241,25 @@ export const formatDataPlaneName = (dataPlaneName: DataPlaneName) => {
     return formattedName.trim();
 };
 
+export const formatDataPlaneOption = ({
```

Comment:
Hmm... I am inclined to suggest calling this function `getDataPlaneOption` as I would expect `formatDataPlaneOption` to receive a data-plane option as an input and return a manipulated/altered data-plane option.